pkg/proxy/proxy.go

```diff
@@ -136,23 +136,44 @@ func generateURL(w http.ResponseWriter, req *http.Request, dport uint16) (*url.U
 		return nil, fmt.Errorf("unable to parse port string: %s", err)
```

Comment:
Missing `Signed-off-by: John Fastabend <john.fastabend@gmail.com>` on this commit